etl/steps/data/garden/faostat/2025-03-17/additional_variables.py

```diff
@@ -1219,6 +1219,65 @@ def generate_net_exports_as_share_of_supply(tb_fbsc: Table) -> Table:
     return tb
 
 
+def generate_milk_per_cow(tb_qcl: Table) -> Table:
+    # FAOSTAT QCL used to have yield for milk (milk per cow), but it was removed at some point (that combination of item and element is empty in the 2025 release).
+    # Numerically, it seems that their old milk yield indicator coincides with production / slaughtered animals.
+    # So I will reproduce that indicator using their latest data on production and number of cows.
+
+    # Firstly, check if yield is still missing in the data.
+    error = "Yield for milk is not missing in the data anymore. Consider reusing it instead of calculating it."
+    assert tb_qcl[(tb_qcl["item_code"] == "00001780") & (tb_qcl["element_code"] == "005420")].empty, error
+
+    # Element code for "Production".
+    ELEMENT_CODE_FOR_PRODUCTION = "005510"
+    # Unit for element of production.
+    UNIT_FOR_PRODUCTION = "tonnes"
+    # Element code for "Milk animals".
+    ELEMENT_CODE_FOR_MILK_ANIMALS = "005318"
+    # Unit for element of animals.
+    UNIT_FOR_ANIMALS = "animals"
+    # Item code for "Raw milk of cattle".
+    ITEM_CODE_FOR_COW_MILK = "00000882"
+
+    # Select the relevant items/elements/units.
+    tb_milk = tb_qcl[
+        (tb_qcl["item_code"] == ITEM_CODE_FOR_COW_MILK)
+        & (tb_qcl["element_code"].isin([ELEMENT_CODE_FOR_PRODUCTION, ELEMENT_CODE_FOR_MILK_ANIMALS]))
+    ].reset_index(drop=True)
+
+    # Sanity check.
+    error = "Units of milk production or milk animals have changed."
+    assert set(tb_milk["unit"]) == {UNIT_FOR_PRODUCTION, UNIT_FOR_ANIMALS}, error
+
+    # Transpose data and rename columns conveniently.
+    tb_milk = tb_milk.pivot(
+        index=["country", "year"], columns="element_code", values="value", join_column_levels_with="_"
+    ).rename(
+        columns={
+            ELEMENT_CODE_FOR_PRODUCTION: "milk_production",
+            ELEMENT_CODE_FOR_MILK_ANIMALS: "cows_used_for_milk",
```

Comment:
I think it's important to note that cattle aren't the main source of milk in many countries e.g. it appears to be camels in Saudi Arabia, in Jamaica I think it is goats. I don' think you can make an equivalence between 'milk animals' and cows.